### ISSUE TITLE
Exception raised when selected a non fitted data file in dataexplorer

### ISSUE THREAD

**Issue by quentinmarolleau** [adareau/HAL#50]
```
Traceback (most recent call last):
  File "/home/quentin/miniconda3/envs/HAL/lib/python3.9/site-packages/pyqtgraph/graphicsItems/ImageItem.py", line 462, in paint
    self.render()
  File "/home/quentin/miniconda3/envs/HAL/lib/python3.9/site-packages/pyqtgraph/graphicsItems/ImageItem.py", line 451, in render
    fn.makeARGB(image, lut=lut, levels=levels, output=self._processingBuffer)
  File "/home/quentin/miniconda3/envs/HAL/lib/python3.9/site-packages/pyqtgraph/functions.py", line 1163, in makeARGB
    raise Exception('levels argument is required for float input types')
Exception: levels argument is required for float input types
Traceback (most recent call last):
  File "/home/quentin/miniconda3/envs/HAL/lib/python3.9/site-packages/pyqtgraph/graphicsItems/ImageItem.py", line 462, in paint
    self.render()
  File "/home/quentin/miniconda3/envs/HAL/lib/python3.9/site-packages/pyqtgraph/graphicsItems/ImageItem.py", line 451, in render
    fn.makeARGB(image, lut=lut, levels=levels, output=self._processingBuffer)
  File "/home/quentin/miniconda3/envs/HAL/lib/python3.9/site-packages/pyqtgraph/functions.py", line 1163, in makeARGB
    raise Exception('levels argument is required for float input types')
Exception: levels argument is required for float input types
Traceback (most recent call last):
  File "/home/quentin/miniconda3/envs/HAL/lib/python3.9/site-packages/pyqtgraph/graphicsItems/ImageItem.py", line 462, in paint
    self.render()
  File "/home/quentin/miniconda3/envs/HAL/lib/python3.9/site-packages/pyqtgraph/graphicsItems/ImageItem.py", line 451, in render
    fn.makeARGB(image, lut=lut, levels=levels, output=self._processingBuffer)
  File "/home/quentin/miniconda3/envs/HAL/lib/python3.9/site-packages/pyqtgraph/functions.py", line 1163, in makeARGB
    raise Exception('levels argument is required for float input types')
Exception: levels argument is required for float input types
Traceback (most recent call last):
  File "/home/quentin/miniconda3/envs/HAL/lib/python3.9/site-packages/pyqtgraph/graphicsItems/ImageItem.py", line 462, in paint
    self.render()
  File "/home/quentin/miniconda3/envs/HAL/lib/python3.9/site-packages/pyqtgraph/graphicsItems/ImageItem.py", line 451, in render
    fn.makeARGB(image, lut=lut, levels=levels, output=self._processingBuffer)
  File "/home/quentin/miniconda3/envs/HAL/lib/python3.9/site-packages/pyqtgraph/functions.py", line 1163, in makeARGB
    raise Exception('levels argument is required for float input types')
Exception: levels argument is required for float input types
Traceback (most recent call last):
  File "/home/quentin/miniconda3/envs/HAL/lib/python3.9/site-packages/pyqtgraph/graphicsItems/ImageItem.py", line 462, in paint
    self.render()
  File "/home/quentin/miniconda3/envs/HAL/lib/python3.9/site-packages/pyqtgraph/graphicsItems/ImageItem.py", line 451, in render
    fn.makeARGB(image, lut=lut, levels=levels, output=self._processingBuffer)
  File "/home/quentin/miniconda3/envs/HAL/lib/python3.9/site-packages/pyqtgraph/functions.py", line 1163, in makeARGB
    raise Exception('levels argument is required for float input types')
Exception: levels argument is required for float input types

```

**Comment by adareau** [comment]
Looks like a duplicate of #44, that was fixed by #45 in devel. This is not fixed in the ROI branch however ; are you still seeing the bug in devel ?

**Comment by quentinmarolleau** [comment]
You're right, I missed that. The bug is fixed already :)
